cpp/include/cudf/scalar/scalar.hpp

```diff
@@ -89,7 +95,10 @@ class scalar {
   rmm::device_scalar<bool> _is_valid;  ///< Device bool signifying validity
 
   scalar() = delete;
-
+  /**
```

Comment:
I would prefer to see a blank line above these docstrings for consistency with most of libcudf. Unfortunately clang-format doesn't appear to have an option for enforcing that blank line. Applies to several places in this PR.
```suggestion

  /**
```